### ISSUE TITLE
[quest]" A Rough Ride" Frenzytribe

### ISSUE THREAD

**Issue by nikofantoma** [dalaranwow/dalaran-wow#5171]
[//]: # (INFORMATION ABOUT TAGS)
[//]: # (The black and yellow tag is unique. There can only be one label assigned to them - one black and one yellow)
[//]: # (Priority - ... tags are assigned by team members)
[//]: # (Your issue will be reviewed by testers and the developers will fix it in the future.)
[//]: # (Only once the issue got the label [Fixed - On Live Server] you can find the fix on your Gameserver.)
[//]: # (You can increase the relevance of your issue by adding a thumb-up emote http://i.imgur.com/RFouVVi.png)

[//]: # (***************************************************************)
[//]: # (** !!!!!!!!!!!!!DON'T DELETE THIS TEMPLATE ELSE YOUR ISSUE WILL BE CLOSED!!!!!!!!!!!! **)
[//]: # (***************************************************************)

**Current behaviour**: Tell us what happens in detail (Where, When, What, Why, Who)
The quest works fine until u have to finish it and return the quest. After you finish your ride with the crock instead of the guy for the quest showing up(cos you need to ride with him) you just got dissmounted and nothing happens further.

**Expected behaviour**: Tell us what should happen instead
There is a quest npc who must shows up and return the quest 
**Steps to reproduce the problem**:

1. 
2. 
3. 

**Include proofs for this behaviour**

You can research this on several pages like http://wowhead.com or http://wowwiki.wikia.com or http://www.youtube.com/
Issues without proofs will be closed and tag On Hold will be added until the proof is provided.

**Include the ID for the game objects, npcs (creatures,pets,minions), spells, items, quests, instances, zones, achievements, skills**

You can research this on several pages like https://wowgaming.altervista.org/aowow/ or https://wowhead.com/

**Include Screenshots from the issue if necessary**

**Comment by TeridanDW** [comment]
Hello and Thank you for the report @nikofantoma,
Issue confirmed on PTR.
Just to clarify when the player is dismounted (at the last waypoint) item https://wowgaming.altervista.org/aowow/?item=38512 or spell https://wowgaming.altervista.org/aowow/?spell=51189 should be used automatically as seen here:
https://www.youtube.com/watch?v=ddRHWMHkli8

Also the crocodile that the player rides should be running not walking thoughtout his path as he does now.

**Comment by TeridanDW** [comment]
Hello again, 
I've been looking in to this issue and I managed to fix it.

 - This:

> Also the crocodile that the player rides should be running not walking thoughtout his path as he does now.

is correct, but after checking the waypoint script, it was already set to run mode. So I've changed his run speed in creature_tempale to match visually the video above.

 - Zepik will now spawn at the end of the quest just like it is shown on the video above.

 - I've also wanted to add the visual effects (chains) from the quest "starter" here:
![image](https://user-images.githubusercontent.com/6797611/33992682-4062fcaa-e0dc-11e7-9ca2-aaed7a928757.png)

and this is how it looks now:
![image](https://user-images.githubusercontent.com/6797611/33992729-67159286-e0dc-11e7-8979-f40c5db43ea0.png)

but if I go away and come back to the area the channeling spell https://wowgaming.altervista.org/aowow/?spell=51256 is nolonger shown on my client even though it's still there:
![image](https://user-images.githubusercontent.com/6797611/33992807-9f693dea-e0dc-11e7-8cf7-ed3a6571e503.png)

In my mind this is a general core issue with all channeling spells and it will be great if someone can look in to it.
Thank you.

**Comment by TeridanDW** [comment]
@Lothloryen the 512s in -- Captive Crocolisk SAI (SET @ENTRY := 28308;) are intended this time as without them the quest credit is not given.
![image](https://user-images.githubusercontent.com/6797611/33993246-684e7b52-e0de-11e7-88b4-bb29b27605ac.png)
